src/EditorFeatures/CSharpTest/Diagnostics/DiagnosticAnalyzerDriver/DiagnosticAnalyzerDriverTests.cs

```diff
@@ -48,10 +48,6 @@ public async Task DiagnosticAnalyzerDriverAllInOne()
             // https://github.com/dotnet/roslyn/issues/44682 - Add to all in one
             missingSyntaxNodes.Add(SyntaxKind.WithExpression);
             missingSyntaxNodes.Add(SyntaxKind.RecordDeclaration);
-            // PROTOTYPE(list-patterns)
```

Comment:
@mavasani Any idea what this test is testing? It looks like it's a copy/paste from src/Compilers/CSharp/Test/Semantic/Diagnostics/DiagnosticAnalyzerTests.AllInOne.cs and it's not clear to me why we're doing the testing at both layers.